scripts/tools/retry.sh

```diff
@@ -9,13 +9,49 @@ function retry {
     local delay="${DELAY:-5}"
     local i=0
 
+    local cr
+    local nl
+    local isa_tty="false"
+    if [ -t 1 ]; then
+        isa_tty="true"
+        # Output is to a TTY; don't scroll display while waiting.
+        cr="\r"
+        # Display a trailing space between the last character and the cursor.
+        nl=" "
+    else
+        # Output is to a file; don't overwrite lines.
+        cr=""
+        nl="\n"
+    fi
+
+    local output
+    output="$(mktemp)"
     while test "${i}" -lt "${max}" ; do
-        printf "%s/%s: %s\\n" "$(( i + 1 ))" "${max}" "$*"
-        if "$@" ; then
+        printf "${cr}[%2dm %2ds] %s/%s: %s${nl}" \
+               "$(( SECONDS / 60 ))" "$(( SECONDS % 60))"\
```

Comment:
Super-nit: inconsistent spacing (space before backslash or not?)

… Nope, I'm not blocking a merge on this.